src/main/java/io/orten/nano/impl/SwishImplementation.java

```
@@ -0,0 +1,165 @@
+package io.orten.nano.impl;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.orten.nano.model.SwishPaymentRequestObject;
+
+import javax.net.ssl.*;
+import java.io.*;
+import java.net.URL;
+import java.net.URLConnection;
+import java.security.KeyStore;
+import java.security.SecureRandom;
+import java.security.cert.X509Certificate;
+import java.util.Map;
+
+
+public class SwishImplementation {
+
+    public static URLConnection createURL(String url) throws Exception{
+        URL myURL = new URL(url);
+        URLConnection connection = myURL.openConnection();
+        connection.setDoOutput(true);
+        return connection;
+    }
+
+    public static void writeObject(InputStream input) throws IOException {
+        BufferedReader in = null;
+        try {
+
+            in = new BufferedReader(new InputStreamReader(input));
+
+            String decodingString;
+            while ((decodingString = in.readLine()) != null) {
+                System.out.println(decodingString);
+            }
+        } finally {
+            if (in != null)
+                in.close();
+        }
+    }
+    public  static  void  saveData(URLConnection con, String data) throws IOException{
+        if(con !=null) {
+            OutputStreamWriter out = new OutputStreamWriter(con.getOutputStream());
+            out.write(data);
+            out.close();
+        }else{
+            System.out.println("Connetion problem !!!");
```

should probably throw an exception here